docs/release-notes.d/00readme.md

```diff
@@ -0,0 +1,28 @@
+# Pending release notes directory
+
+This directory contains release note entries that have not been merged
+into the main release-notes.md document.
+
+Generally a release note entry should be created for changes that:
+
+- Fix bugs in the code.
+- Implement new features.
+- Change existing behavior.
+
+Release notes are generally not needed for:
+
+- Documentation improvements.
```

Comment:
Sometimes documentation improvement may require release note, for example if documentation states some MTU and it is incorrect, by mistake, and code has the right one, the documentation will be corrected which requires external code to follow.